config/initializers/session_store.rb

```diff
@@ -1,9 +1,16 @@
 service_url = ENV['SERVICE_URL']
 SERVICE_DOMAIN = (service_url) ? URI.parse(service_url).host : nil
+timeout_time = ENV['TIMEOUT_TIME']
```

Comment:
`TIMEOUT_TIME` is a bit redundant.  I'd prefer just `timeout` for the variable name and something more specific for the environment variable, `PVB_SESSION_TIMEOUT`, perhaps.